config/application.rb

```diff
@@ -26,6 +26,8 @@ class Application < Rails::Application
     # generic overrides
     config.to_prepare do
```

Comment:
As we are already using initializers files, let's put all of it in initialzer files like the "custom_menu.rb".
Instead of custom_menus we could use somthing like "conferences_hacks.rb" or simiar.

on the other hand the controlleroverrides better do it in a initializer after the other overrides:

```ruby
initializer "capitalitat.controller_overrides", after: "decidim.action_controller" do
      config.to_prepare do
        Decidim::Conference.include(ConferenceOverride)
      end
    end
    
    initializer "capitalitat.homepage_content_blocks" do
```